driver/src/contracts/snapp_contract.rs

```diff
@@ -20,11 +20,13 @@ pub struct SnappContractImpl {
 
 impl SnappContractImpl {
     pub fn new() -> Result<Self> {
-        let contract_json = fs::read_to_string("dex-contracts/build/contracts/SnappAuction.json").unwrap();
-        let address = env::var("SNAPP_CONTRACT_ADDRESS").unwrap();
-        Ok(SnappContractImpl {
-            base: BaseContract::new(address, contract_json).unwrap()
-        })
+        let contract_json = fs::read_to_string("dex-contracts/build/contracts/SnappAuction.json");
```

Comment:
nit: let's put the ? on this line so that the variables are already the correct success type.

Reply:
Ahh! ok will do.